torchvision/models/detection/faster_rcnn.py

```diff
@@ -347,7 +347,7 @@ def fasterrcnn_resnet50_fpn(pretrained=False, progress=True,
         trainable_backbone_layers (int): number of trainable (not frozen) resnet layers starting from final block.
             Valid values are between 0 and 5, with 5 meaning all backbone layers are trainable.
     """
-    assert trainable_backbone_layers <= 5 and trainable_backbone_layers >= 0
+    assert 0 <= trainable_backbone_layers <= 5
```

Comment:
I understand why you would want to fix this and I'm not opposed to it in general. But this is out of scope for this PR. If you want to make that change we are happy to accept a follow-up PR after this one is merged.